psydac/feec/derivatives.py

```diff
@@ -86,6 +257,15 @@ def domain(self):
     @property
     def codomain(self):
         return self._codomain
+    
+    def __call__(self, u):
+        assert isinstance(u, FemField)
+        assert u.space == self.domain
+
+        coeffs = self.matrix.dot(u.coeffs)
+        coeffs.update_ghost_regions()
```

Comment:
I'll try and see, if it fails somewhere... A question then: should the `update_ghost_regions` in line 176 then also be removed? (since it is a post-operation update as well)

Reply:
It doesn't seem to fail the tests which are directly related to it; so I removed both the `update_ghost_regions` in 266 and in 176.